src/components/BreakoutRoomsEditor/SendMessageDialog.vue

```diff
@@ -111,4 +123,9 @@ export default {
 		margin-bottom: 0;
 	}
 }
+
+:deep(.modal-container) {
+	overflow: visible !important;
```

Comment:
Please leave a comment what gets fixed by this, so we know when removing what should be tested :)